tests/cypress/tests-e2e/search/search-results.cy.js

```diff
@@ -466,5 +466,18 @@ describe('Test the search bar result handling', () => {
             expect(feature[0]).to.be.approximately(coordinates[0], acceptableDelta)
             expect(feature[1]).to.be.approximately(coordinates[1], acceptableDelta)
         })
+
+        cy.get('[data-cy="search-results-locations"] [data-cy="search-result-entry"]')
+            .as('locationSearchResults')
+            .first()
+            .invoke('text')
+            .then((text) => text.trim())
+            .should('eq', '1530 Payerne')
+
+        // For https://jira.swisstopo.ch/browse/PB-400
```

Comment:
to be fair, I don't think anyone will check the Jira tickets. I would add the following at the very beginning of the whole section you added : 

```
// ----------------------------------------------------------------------
cy.log('Ensuring the search dialog closes once you've selected an item')
cy.get('[data-cy ...
```
That way, we make a separation between this test case and the previous ones.

Reply:
ah yes, that's actually correct. We can still also go to the ticket by checking the commit message or PR on the line of code.